### DISCUSSION TITLE
Is there a demo application code running on top of this adaptive stack?

### DISCUSSION THREAD

**Discussion by mathiga** [langroodi/Adaptive-AUTOSAR#13] [Q&A]
Hi , I would like to see how the application uses the ara::com and ara::exec component which is implemented in this project , I am interested to see what tool we can use the get the service interface definition to communicate between apps

**Comment by langroodi** [comment]
Hi! Yes, there will be a demo which first covers ara::diag, SM, and ara::exec which is planned to be finished on December this year and another demo will be available to also cover ara::com which is planned to be finished on May next year. That latest demo shows how an Adaptive Application will interact with other AUTOSAR components.
